### PR TITLE
Update helm chart for OES 3.8.1

### PR DESCRIPTION
**Pull request by girichinna27**
Please review the changes

### CODE REVIEW

**Comment by abhinaybyrisetty** [comment]
Images should be tagged as <major>.<minor>.<patch>

Even last time when images were provided, I retagged 3.8 images as 3.8.0, updated the chart accordingly and published.

Please correct it. Also, I noticed that gate repository name has been changed as ubi8-gate, why? Is this the common gate for both Spinnaker and OES in house services?

File: charts/oes/values.yaml [hunk 1]
```diff
@@ -283,7 +283,7 @@ dashboard:
   image:
     registry: quay.io/opsmxpublic
     repository: ubi8-oes-dashboard
-    tag: v3.8.0
+    tag: v3.8
```
Comment (abhinaybyrisetty) [inline charts/oes/values.yaml:286]: why did we change this? it should be v3.8.x

File: charts/oes/values.yaml [hunk 2]
```diff
@@ -560,7 +560,7 @@ visibility:
   image:
     registry: quay.io/opsmxpublic
     repository: ubi8-oes-visibility
-    tag: v3.8.0
+    tag: v3.8
```
Comment (abhinaybyrisetty) [inline charts/oes/values.yaml:563]: why did we change this? it should be v3.8.x

File: charts/oes/values.yaml [hunk 3]
```diff
@@ -332,8 +332,8 @@ gate:
   ##
   image:
     registry: quay.io/opsmxpublic
-    repository: ubi8-oes-gate
-    tag: v3.8.0
+    repository: ubi8-gate
```
Comment (abhinaybyrisetty) [inline charts/oes/values.yaml:335]: why did we change this?

**Review by abhinaybyrisetty — CHANGES_REQUESTED** [review]

File: charts/oes/values.yaml (hunk 1 above)
Comment (girichinna27) [reply charts/oes/values.yaml]: fixed it

File: charts/oes/values.yaml (hunk 2 above)
Comment (girichinna27) [reply charts/oes/values.yaml]: fixed it

File: charts/oes/values.yaml (hunk 3 above)
Comment (girichinna27) [reply charts/oes/values.yaml:335]: it's common gate; as per discussion with Jay & developer we have removed the  OES from image name.

Comment (abhinaybyrisetty) [reply charts/oes/values.yaml:335]: Don't we need any other changes in the helm chart to accomodate common gate?

**Review by abhinaybyrisetty — APPROVED** [review]
Ok

**Pull request merged**